frontend/package.json

```diff
@@ -52,7 +52,7 @@
     "@patternfly/react-icons": "4.93.6",
     "@patternfly/react-styles": "4.92.6",
     "@patternfly/react-table": "4.112.39",
-    "@patternfly/react-topology": "^4.91.27",
+    "@patternfly/react-topology": "^5.0.0",
```

Comment:
We are at [5.1.0](https://github.com/opendatahub-io/odh-dashboard/blob/845160368eae5fff39fc282c36d99a195f24b7d7/frontend/package.json#L55)